install/files/local/env

```diff
@@ -23,7 +23,7 @@ SESSION_LIFETIME=120
 
 MEMCACHED_HOST=cache
 
-MAIL_MAILER=smtp
+MAIL_MAILER=log
 MAIL_HOST=smtp.mailtrap.io
 MAIL_PORT=2525
 MAIL_USERNAME=null
```

Comment:
Since `local` has stubbed emails, you could set the rest of these `MAIL_*` variables to be empty.

Reply:
I think I'll just leave them alone for now, just so that the various .env files are only minimally modified. Makes it easier to diff them.